test/NuGet.Core.Tests/NuGet.Commands.Test/SpecValidationUtilityTests.cs

```diff
@@ -53,6 +56,11 @@ public void SpecValidationUtility_VerifyFrameworks_Unsupported()
             // Arrange
             var spec = new DependencyGraphSpec();
             spec.AddRestore("a");
+            var errors = new List<string>();
```

Comment:
Just make it a List of error codes and avoid the ToString and Contains overhead?